editor/import/resource_importer_csv_translation.cpp

```diff
@@ -54,7 +54,7 @@ String ResourceImporterCSVTranslation::get_save_extension() const {
 
 String ResourceImporterCSVTranslation::get_resource_type() const {
 
-	return "StreamCSVTranslation";
+	return "Translation";
```

Comment:
Mhh well yeah, but every csv is a translation (at least in this context). The way I see it is that this class is _only_ used for importing csv translations. It goes (more or less) like this:

- User requested to load a Translation. Say, from the property inspector.
- The engine loops through all the `ResourceImporter`s and calls `get_resource_type()` on them, looking for `Translation`. It'll find this class.
- The file dialog opens and the filter is set to *.csv (`get_recognized_extensions()`
- On file selection, [`import()`](https://github.com/Hinsbart/godot/blob/4d8e45de17ce548601b5a9ef474197eff098a576/editor/import/resource_importer_csv_translation.cpp#L78) will be called, saving a `Ref<Translation>`.

Soo.. I think it's fine ^^